test/classes/initializers/errors/errHandling/initWithTryBang.chpl

```diff
@@ -0,0 +1,22 @@
+// Verifies we currently don't allow initializers to have try! statements in
+// their body.
+// We hope to eventually allow it.
+class Foo {
+  var x: int;
+
+  proc init() {
+    x = 10;
+    try! {
+      outerFunc();
+    }
+    super.init();
+  }
+}
+
+proc outerFunc() throws {
+  throw new Error();
+}
+
+var foo = new Foo();
```

Comment:
@mppf - without the changes on this branch, this test generates the output:
```
uncaught Error:
   initWithTryBang.chpl:17: thrown here
   initWithTryBang.chpl:20: uncaught here
```

Reply:
That seems just fine.

Reply:
Shouldn't the line number be within the 9-11 range, since it is the `try!` that didn't catch the error rather than the `var foo = new Foo();`?

Reply:
Yes, the second line number ought to be 9-11 and not 20. I'm not sure what's going wrong there, but would guess it's simply some mis-attributed line numbers on AST elements in the compiler.

Reply:
Should I accept the currently generated numbers for this commit, even though it is less than ideal?  Or should I fix that now (or in a separate PR)?

Reply:
I certainly don't think you need to do it in this PR. I think it's worth looking in to at some point but it doesn't have to happen before Monday.

Reply:
If the line numbers are wrong for this test, shouldn't there be a `.bad` file for it? Or did I just miss that?

Reply:
If the line numbers are wrong for this test, shouldn't there be a `.future` and `.bad` file for it? Or did I just miss that?

Reply:
I decided to make only one of the three a future, with a note to update the other two when it gets fixed